Clarifications in job-limits file (#4984)
Qiskit/documentation: docs/guides/job-limits.mdx

---
# Job limits

> When you send a job to an IBM&reg; QPU, it is first validated to try to ensure that the job can run on the QPU, so you don't have to wait for it to go through the queue, only for the job to fail at that point. These checks include enforcing the following described limits. If these limits are exceeded, that workload cannot be handled by the quantum software stack and will usually fail.

Author: ```suggestion
When you send a job to an IBM&reg; QPU, it is first validated to try to ensure that the job can run on the QPU, to decrease the likelihood that your job fails after you've waited for it to go through the queue. These checks include enforcing the following described limits. If these limits are exceeded, that workload cannot be handled by the quantum software stack and will usually fail.
```
